_i18n/en/resources/user-guides/cli_wallet_daemon_isolation_qubes_whonix.md

```diff
@@ -13,32 +13,14 @@ This is safer than other approaches which route the wallets rpc over a Tor hidde
 
   - The first workstation will be used for your wallet, it will referred to as `monero-wallet-ws`. You will have `NetVM` set to `none`.
 
-  - The second workstation will be for the `monerod` daemon, it will be referred to as `monerod-ws`. You will have `NetVM` set to the Whonix gateway `sys-whonix`.
+  - The second workstation will be for the `monerod` daemon, it will be referred to as `monerod-ws`. You will have `NetVM` set to the Whonix gateway `sys-whonix`. Before moving on, make sure this workstation has enough private storage. You can estimate how much space you need by checking the size of the [raw blockchain](https://web.getmonero.org/downloads/#blockchain). Keep in mind that the blockchain will take up more space with time.
```

Comment:
Please change to the link to:
```suggestion
  - The second workstation will be for the `monerod` daemon, it will be referred to as `monerod-ws`. You will have `NetVM` set to the Whonix gateway `sys-whonix`. Before moving on, make sure this workstation has enough private storage. You can estimate how much space you need by checking the size of the [raw blockchain](https://{{ site.baseurl }}/downloads/#blockchain). Keep in mind that the blockchain will take up more space with time.
```
This could be a useful trick to check the size of the blockchain, but keep in mind that that snapshot could be a bit back the current height. Still good to get an idea of the size though.